api/ipfsproxy/ipfsproxy.go

```diff
@@ -647,6 +651,50 @@ func (proxy *Server) repoStatHandler(w http.ResponseWriter, r *http.Request) {
 	return
 }
 
+type ipfsRepoGCResp struct {
+	Key   cid.Cid `json:",omitempty"`
+	Error string  `json:",omitempty"`
+}
+
+func (proxy *Server) repoGCHandler(w http.ResponseWriter, r *http.Request) {
+	queryValues := r.URL.Query()
+	streamErrors := queryValues.Get("stream-errors")
+	// TODO: Does `quiet` really quiet anything?
```

Comment:
Yes, but only for text output https://github.com/ipfs/go-ipfs/blob/master/core/commands/repo.go#L121

Reply:
So we can ignore